channels/slack/src/main/kotlin/com/justai/jaicf/channel/slack/SlackChannel.kt

```diff
@@ -99,7 +112,7 @@ class SlackChannel private constructor(
 
     companion object : JaicpCompatibleAsyncChannelFactory {
         override val channelType = "slack"
-        override fun create(botApi: BotApi, apiUrl: String) = SlackChannel(botApi, apiUrl)
+        override fun create(botApi: BotApi, apiUrl: String) = SlackChannel(botApi, "$apiUrl/")
```

Comment:
I'd append slash inside constructor, incapsulating this logic and keeping factory clean